### ISSUE TITLE
Compilation insruction?

### ISSUE THREAD

**Issue by abylikhsanov** [xiaoxifuhongse/People-Detection-and-Tracking#2]
Hi, I cannot see any build type. Does it depend on the other project?

**Comment by xiaoxifuhongse** [comment]
I just upload source header and cpp files。as a reference ，the paper ，matlab code，you can get your own project

**Comment by abylikhsanov** [comment]
Yes I used CMake to complete the build. For some reason, I am getting a triangle and two circles but no image as you have on a video, do you know why?
